docs/functional/cart.md

```diff
@@ -0,0 +1,37 @@
+# Cart
+This article describes functional behavior of a cart. Article shortly summarize its discount codes, behavior of cart content when user logs in and others.
+
+## Add to cart
+- a user can add the product to cart from the product detail or from the product listing (except for the product with variants)
+- after adding the product to the cart, a pop up window is displayed. There is information about adding the product to the cart together with the button “Go to cart”
+## Cart overview in header
+- shows the amount and total price with VAT of products in the cart
+- shows the information about empty cart
+- changes in content of cart in header are modified by Ajax
+## Page cart
+### URL
+- URL has format youreshopurl/cart (in English version)
+### The cart can be in these states:
+- **empty** - nothing was added to the cart yet, cart shows an image of empty cart and  the customer is informed that the cart is empty
+- **contains products** - the customer can see a list of products which have been added to the cart, customer can change quantity of products or remove product from the cart - these changes are modified by Ajax
+### Price
+- total price including VAT is automatically recalculated (without page refresh) if the product is removed or the amount is changed
+### Discount coupon
```

Comment:
Do we use `Discount coupon` or `Promo code`?

Reply:
I changed discount code to promo code. Thanks